keyboards/malevolti/superlyra/rev1/info.json

```diff
@@ -0,0 +1,101 @@
+{
+    "keyboard_name": "superlyra",
+    "url": "",
+    "maintainer": "malevolti",
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+              { "x": 4, "y": 0 },
```

Comment:
Keys are in the wrong order. Press Ctrl+Shift+I in the Configurator and turn on fast input.

Reply:
Could I possibly have more direction on what I should do here? I originally generated this layout in KLE and then ran the output through https://qmk.fm/converter/.

Reply:
??

Reply:
Did you try what I suggested?

Reply:
how do I access this keyboard on the configurator? this is where I'm confused; I go onto the configurator and select fast input in the settings but from there you've lost me as I see no way of accessing this particular file

Reply:
> Press Ctrl+Shift+I in the Configurator and turn on fast input

Doing this will allow you to load a JSON file.  Which you want to load this json file.

Fast input will show the order that each key is in, and it should match what the layout macro has.

Reply:
Ok, have managed to get that upload dialog up; the hotkey is very hit&miss whether it will trigger on my side of things which was why I was so confused as it wasnt doing anything previously. Can I just confirm that I am working left->right then row-by-row on the configuratior to get the .json into the correct order?

Reply:
tis done